### PR TITLE
add test case

### PR DESCRIPTION
**Pull request by Zero-Rock**
test: add test case

test: remove test snap

test: add test case

test: add test case

## What type of PR is this?

- [ ] Feature
- [ ] Bugfix
- [X] Test
- [ ] Documentation
- [ ] Refactor
- [ ] Style
- [ ] Chore

## What this PR does / why we need it:

add test case for common components

## Which issue(s) this PR fixes:

Fixes #

## Does this PR introduce a user interface change?

- [ ] Yes(screenshot is required)
- [X] No

### CODE REVIEW

File: shell/app/common/__tests__/components/addon-settings.test.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+      expect(wrapper.find('.settings-delete')).not.toExist();
+      wrapper.setProps({
+        addonConfig: {
+          config,
+          canDel: true,
+        },
+      });
+      wrapper.update();
+      expect(wrapper.find('.settings-delete')).toExist();
+      wrapper.find('[deleteItem="service"]').prop('onConfirm')();
```
Comment (daskyrk) [inline shell/app/common/__tests__/components/addon-settings.test.tsx:105]: Is this line useless?

File: shell/app/common/__tests__/components/key-value-table.test.tsx [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    expect(editor.state().dataSource).toHaveLength(4);
+    editor.find('Popconfirm').at(0).prop('onConfirm')();
+    expect(editor.state().dataSource).toHaveLength(3);
+    wrapper.setProps({
+      data: { ...data, displayName: 'erda', userName: 'erda', CPU: '2' },
+    });
+    wrapper.update();
+    expect(editor.state().dataSource).toHaveLength(6);
+  });
+  it('should should render with Input', () => {
```
Comment (daskyrk) [inline shell/app/common/__tests__/components/key-value-table.test.tsx:82]: duplicate should

File: shell/app/common/__tests__/components/__snapshots__/loading-skeleton.test.tsx.snap [hunk 3]
```diff
@@ -0,0 +1,41 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (daskyrk) [inline shell/app/common/__tests__/components/__snapshots__/loading-skeleton.test.tsx.snap:1]: Is this snap file useful?

Comment (Zero-Rock) [reply shell/app/common/__tests__/components/__snapshots__/loading-skeleton.test.tsx.snap:1]: this component without events and data，the rendering result is just a Dom tree

**Review by daskyrk — APPROVED** [review]

**Pull request merged**